static/app/components/charts/barChart.tsx

```diff
@@ -10,37 +10,31 @@ type ChartProps = Omit<React.ComponentProps<typeof BaseChart>, 'css'>;
 
 export type BarChartSeries = Series & Omit<BarSeriesOption, 'data' | 'name'>;
 
-type Props = Omit<ChartProps, 'series'> & {
+export interface BarChartProps extends Omit<ChartProps, 'series'> {
   series: BarChartSeries[];
   animation?: boolean;
   stacked?: boolean;
-};
-
-class BarChart extends React.Component<Props> {
-  render() {
-    const {series, stacked, xAxis, animation, ...props} = this.props;
-
-    return (
-      <BaseChart
-        {...props}
-        xAxis={xAxis !== null ? {...(xAxis || {}), boundaryGap: true} : null}
-        series={series.map(({seriesName, data, ...options}) =>
-          BarSeries({
-            name: seriesName,
-            stack: stacked ? 'stack1' : undefined,
-            data: data.map(({value, name, itemStyle}) => {
-              if (itemStyle === undefined) {
-                return [name, value];
-              }
-              return {value: [name, value], itemStyle};
-            }),
-            animation,
-            ...options,
-          })
-        )}
-      />
-    );
-  }
 }
 
-export default BarChart;
+export function BarChart({series, stacked, xAxis, animation, ...props}: BarChartProps) {
+  return (
+    <BaseChart
+      {...props}
+      xAxis={xAxis !== null ? {...(xAxis || {}), boundaryGap: true} : null}
+      series={series.map(({seriesName, data, ...options}) =>
+        BarSeries({
+          name: seriesName,
+          stack: stacked ? 'stack1' : undefined,
+          data: data.map(({value, name, itemStyle}) => {
+            if (itemStyle === undefined) {
+              return [name, value];
```

Comment:
The two branches are returning different shape objects, is this expected? (I see it was doing that before already, so probably fine)

Reply:
I did not change this code... it was just moved